apps/TutorialServo/TutorialServo.cpp

```diff
@@ -0,0 +1,42 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "TutorialServo.h"
+#include "mbed.h"
+
+
+void TutorialServo::setAngleRangeInDegrees(float degrees)
+{
+  angleRange = degrees;
+}
+
+void TutorialServo::setPulseWidthRangeInMs(float minPulseWidthMsFloat, float maxPulseWidthMsFloat)
+{
+  minPulseWidthMs = minPulseWidthMsFloat;
+  maxPulseWidthMs = maxPulseWidthMsFloat;
+}
+
+void TutorialServo::setPositionInDegrees(float degrees)
+{
```

Comment:
Usually, a servo can rotate in both directions, this means the setPositionInDegrees should be able to take in negative argument. Does the current logic still work in case degrees is negative? You would also need to define what angleRange means, does it represent the absolute value of the maximum angle or the full range from maximum angle in positive direction to maximum angle in negative direction ?

Reply:
Okay, it is now accounted for!